xml-compare/src/main/java/utils/ZipUtils.java

```diff
@@ -0,0 +1,64 @@
+package utils;
+
+import lombok.NonNull;
+import lombok.extern.log4j.Log4j2;
+import org.apache.commons.io.IOUtils;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Enumeration;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+@Log4j2
+public class ZipUtils {
+
+    public static void extractGoldDataAndOutputData(@NonNull File zipFile, @NonNull String goldDataFolder,
+                                                    @NonNull String outputDataFolder, @NonNull String goldDataRegex,
+                                                    @NonNull String outputDataRegex) {
+
+        if (!zipFile.exists()) log.error("File '{}' does not exist", zipFile);
```

Comment:
Java Code Convention prefers use braces for if statements even in this short line for better code :)
I also suggest you to try to use maven-checkstyle-plugin. It really helps to find some code style violations and fix it easily. For your code it is about 40 violations (using shoggy's checkstyle configuration).

Reply:
Fixed)
Well, don't really like all of the rules in there, but I fixed most of the plugin comments